substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/DefaultOptionHandler.java

```diff
@@ -198,6 +201,14 @@ public boolean consume(ArgumentQueue args) {
                 nativeImage.addPlainImageBuilderArg("-H:DiagnosticsDir=" + nativeImage.diagnosticsDir);
                 System.out.println("# Diagnostics mode enabled: image-build reports are saved to " + nativeImage.diagnosticsDir);
                 return true;
+            case "--list-cpu-features":
+                args.poll();
+                nativeImage.showMessage("Available AMD64 CPUFeatures: " + Arrays.toString(AMD64.CPUFeature.values()));
```

Comment:
Wouldn't it make sense to only show the feature set for the current arch, at least until arch cross-compilation is supported?

Reply:
Yes, I could base this on System.getProperty("os.arch") to only show the host architecture. To me there doesn't seem to be any harm in showing information about both architecture. If you feel strongly about this though I'm happy to change it

Reply:
We should just be consistent with the rest of Native Image help. Do we take this all-platform approach elsewhere? And as you say, does it matter? I'm no expert on this so would like to see if @christianwimmer or @olpaw have an opinion.